tgui/packages/tgui/interfaces/AirAlarm.tsx

```diff
@@ -212,6 +219,16 @@ const AirAlarmControlHome = (props, context) => {
         content="Alarm Thresholds"
         onClick={() => setScreen('thresholds')}
       />
+      {!!sensor && (
+        <Box mt={1}>
```

Comment:
Should(*) be able to remove this box and put mt=1 on button

Reply:
No, this is a block wrapper for the button. Button is an inline box, it will be displayed on the same line as the previous button. If I make it fluid, it will take the whole width. This box is there just for the line break.